pkg/kubelet/kuberuntime/kuberuntime_manager_test.go

```diff
@@ -1170,6 +1170,17 @@ func TestComputePodActions(t *testing.T) {
 				ContainersToStart: []int{1},
 			},
 		},
+		"Restart the container if the container is in created state": {
```

Comment:
I see, you're adding a test case here because the kubelet handles the Regular Containers [the same way](https://github.com/kubernetes/kubernetes/blob/00236ae0d73d2455a2470469ed1005674f8ed61f/pkg/kubelet/container/helpers.go#L89-L92).